### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by bwaj287**
[release] core_java project

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Compiled source #
```
Comment (jarviscanada) [inline .gitignore:1]: wrong PR title...please put [release] prefix next time

File: core_java/grep/grep.iml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
+  <component name="CheckStyle-IDEA-Module">
+    <option name="configuration">
```
Comment (jarviscanada) [inline core_java/grep/grep.iml]: git rm this file

File: core_java/grep/src/main/java/ca/jrvs/apps/practice/HelloWorld.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package ca.jrvs.apps.practice;
+
+public class HelloWorld {
+  // Your program begins with a call to main().
```
Comment (jarviscanada) [inline core_java/grep/src/main/java/ca/jrvs/apps/practice/HelloWorld.java:4]: remove

File: core_java/jdbc/jdbc.iml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
+  <component name="CheckStyle-IDEA-Module">
+    <option name="configuration">
+      <map />
```
Comment (jarviscanada) [inline core_java/jdbc/jdbc.iml]: git rm

File: core_java/twitter/src/main/java/ca/jrvs/apps/twitter/dao/TwitterDao.java [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+  protected Tweet parseResponseBody(HttpResponse response, int expectedStatusCode) {
+    Tweet tweet = null;
+
+
+    int status = response.getStatusLine().getStatusCode();
+    if (status != expectedStatusCode) {
+      try {
+        System.out.println(EntityUtils.toString(response.getEntity()));
+      } catch (IOException e) {
+        System.out.println("no Response");
```
Comment (jarviscanada) [inline core_java/twitter/src/main/java/ca/jrvs/apps/twitter/dao/TwitterDao.java:128]: use the logger instead.. logger.error

Comment (jarviscanada) [reply core_java/twitter/src/main/java/ca/jrvs/apps/twitter/dao/TwitterDao.java:128]: fix this for all occurrences. don't use println except for printing tweet object

File: core_java/twitter/twitter.iml [hunk 6]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
+  <component name="CheckStyle-IDEA-Module">
```
Comment (jarviscanada) [inline core_java/twitter/twitter.iml]: git rm this file

File: practice/practice.iml [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    </option>
+  </component>
+  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_1_8">
+    <output url="file://$MODULE_DIR$/target/classes" />
+    <output-test url="file://$MODULE_DIR$/target/test-classes" />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/main/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test/java" isTestSource="true" />
+      <excludeFolder url="file://$MODULE_DIR$/target" />
+    </content>
```
Comment (jarviscanada) [inline practice/practice.iml:15]: `git rm  practice/target/`
don't commit generated files..
add target to your gitignore 

git rm all *.iml files

**Review by jarviscanada — APPROVED** [review]

**Comment by bwaj287** [comment]
core_java project finished

**Pull request closed without merging**